### PR TITLE
PCAmixdata & factoextra

### PR DESCRIPTION
**Pull request by orbsmiv**
For INC1047995

* [x] Assigned to reviewer

Rebuild meta module...
`BEAR-R-bio-2020a-foss-2020a-R-4.0.0.eb --force`
* [ ] EL7-cascadelake
* [ ] EL7-haswell
* [ ] Ubuntu16 VM

### CODE REVIEW

File: easybuild/easyconfigs/f/factoextra/factoextra-1.0.7-foss-2020a-R-4.0.0.eb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    'source_tmpl': '%(name)s_%(version)s.tar.gz',
+}
+
+
+# Order is important!
+exts_list = [
+    ('rstatix', '0.6.0', {
+        'checksums': ['ebb28e20c7e28809194a2a027bc83303b17be1e3db32f49325727c9279df9c5b'],
+    }),
+    ('ggsci', '2.9', {
```
Comment (branfosj) [inline easybuild/easyconfigs/f/factoextra/factoextra-1.0.7-foss-2020a-R-4.0.0.eb:38]: This version is in he R EC.

File: easybuild/easyconfigs/f/factoextra/factoextra-1.0.7-foss-2020a-R-4.0.0.eb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        'checksums': ['4af14e6f3657134c115d5ac5e65a2ed74596f9a8437c03255447cd959fe9e33c'],
+    }),
+    ('ggsignif', '0.6.0', {
```
Comment (branfosj) [inline easybuild/easyconfigs/f/factoextra/factoextra-1.0.7-foss-2020a-R-4.0.0.eb:41]: This version is in he R EC.

File: easybuild/easyconfigs/f/factoextra/factoextra-1.0.7-foss-2020a-R-4.0.0.eb [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        'checksums': ['6fe13efda31386483e64d466ba2f5a53a2a235ae04f5c17bba3ccc63d283499e'],
+    }),
+    ('ggpubr', '0.4.0', {
```
Comment (branfosj) [inline easybuild/easyconfigs/f/factoextra/factoextra-1.0.7-foss-2020a-R-4.0.0.eb:44]: Is 0.4.0 needed? 0.3.0 is in the R EC.

File: easybuild/easyconfigs/f/factoextra/factoextra-1.0.7-foss-2020a-R-4.0.0.eb [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        'https://cran.r-project.org/src/contrib/',  # current version of packages
+        'https://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz',
+}
+
+
+# Order is important!
+exts_list = [
+    ('rstatix', '0.6.0', {
```
Comment (branfosj) [inline easybuild/easyconfigs/f/factoextra/factoextra-1.0.7-foss-2020a-R-4.0.0.eb:35]: Is 0.6.0 needed? 0.5.0 is in the R EC.

Comment (orbsmiv) [reply easybuild/easyconfigs/f/factoextra/factoextra-1.0.7-foss-2020a-R-4.0.0.eb]: It was, but for the updated version of `ggpubr` which in fact isn't necessary under R 4.0.0!

**Pull request merged**